variables.tf

```diff
@@ -96,6 +96,19 @@ variable "continuous_backup_recovery_window_days" {
   default     = 14
 }
 
+variable "maintenance_update_policy" {
+  description = "defines the policy for system updates"
+  type = object({
+    maintenance_windows = optional(object({
+      day = optional(string)
+      start_time = optional(object({
+        hours = optional(number)
```

Comment:
@robertusnegoro it seems like `day` and `start_time` and hours are required field.